app/models/classification.rb

```diff
@@ -23,6 +23,18 @@ class Classification < ActiveRecord::Base
   scope :created_by, -> (user) { where(user: user) }
   scope :complete, -> { where(completed: true) }
 
+  after_create do
```

Comment:
Mostly because it's an enormous pain to call a service object instead of creating AR records with our current JSONApi stuff. Maybe I should refactor one or two controllers and show how I'd do it without all that (in a fresh branch).